extras/kinesis-asl/src/main/java/org/apache/spark/examples/streaming/JavaKinesisWordCountASL.java

```diff
@@ -131,7 +131,7 @@ public static void main(String[] args) {
         for (int i = 0; i < numStreams; i++) {
           streamsList.add(
             KinesisUtils.createStream(jssc, streamName, endpointUrl, checkpointInterval, 
-            InitialPositionInStream.LATEST, StorageLevel.MEMORY_AND_DISK_2())
+            InitialPositionInStream.LATEST, StorageLevel.MEMORY_AND_DISK_2(), null)
```

Comment:
also, might want to name this variable to be more explicit.  you can set it to None/null, but naming params goes a long way in these examples.